src/njoy21/input/PLOTR/Card3a/test/Card3a.test.cpp

```diff
@@ -0,0 +1,23 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "bugless" ){
+  std::string value(
+      "123456789 123456789 123456789 123456789 123456789 123456789 ");
+  iRecordStream<char> iss{ std::istringstream( "'" + value + "'" ) };
+  PLOTR::Card3a card3a( iss );
+  REQUIRE( card3a.title.value == value );
+}
+
+SCENARIO( "bugged" ){
+  GIVEN( "a string that's too long" ){
+    std::string value(
+      "123456789 123456789 123456789 123456789 123456789 123456789 1");
+    iRecordStream<char> iss{ std::istringstream( "'" + value + "'" ) };
+    REQUIRE_THROWS( PLOTR::Card3a( iss ) );
+  }
+}
```

Comment:
I've checked through Card3a. (Including Cards 10–13). I'll pick up on this tomorrow.